leptonica/PSPBUILD

```diff
@@ -0,0 +1,36 @@
+pkgname=leptonica
+pkgver=1.84.1
+pkgrel=1
+pkgdesc="An open source C library for efficient image processing and image analysis operations"
+arch=('mips')
+url="http://www.leptonica.org/"
+license=('BSD-2-Clause')
+depends=('zlib' 'libpng' 'jpeg')
+makedepends=()
+optdepends=()
+source=("https://github.com/DanBloomberg/leptonica/releases/download/${pkgver}/$pkgname-${pkgver}.tar.gz")
+sha256sums=("2b3e1254b1cca381e77c819b59ca99774ff43530209b9aeb511e1d46588a64f6")
+
+prepare() {
+    cd "$pkgname-${pkgver}"
+    sed -i 's#@prefix@#${PSPDEV}/psp#' lept.pc.in
```

Comment:
It seems this one is not working. I'm seeing this in `lept_Release.pc`:
```
prefix=/__w/psp-packages/psp-packages/leptonica/pkg/leptonica/psp
```
Not sure why it's called that either.

Reply:
Maybe their CMake is busted, I'll change it to autotools